Change nomenclature for Roborock fan speeds

tiagofreire-pt opened the pull request:
## Breaking Change:

May break some automations using the string/names from the current implementation.

> Users will need to update any automations that utilize fan speed values – specifically:
> 
>     * `Quiet -> Silent`
> 
>     * `Balanced -> Standard`
> 
>     * `Turbo -> Medium`
> 
>     * `Max -> Turbo`

## Description:
Those are the new fan speed nomenclatures on the Xiaomi app vs 5.6.34.

**Pull request with documentation for [home-assistant.io](https://github.com/home-assistant/home-assistant.io) (if applicable):**
https://github.com/home-assistant/home-assistant.io/pull/11523

## Example entry for `configuration.yaml` (if applicable):
Not applicable.

## Checklist:
  - [X] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [X] There is no commented out code in this PR.
  - [X] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [X] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [x] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [x] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [x] Untested files have been added to `.coveragerc`.

If the code does not interact with devices:
  - [x] Tests have been added to verify that the new code works.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

tiagofreire-pt committed:
New nomenclature for Roborock fan speeds

Those are the new fan speed nomenclatures on the Xiaomi app vs 5.6.34.

ochlocracy committed:
Add support for input_number entities in Alexa integration (#30139)

* Add support for input_number entities

* Update homeassistant/components/alexa/capabilities.py

Co-Authored-By: Paulus Schoutsen <paulus@home-assistant.io>

* Removed get methods to directly access required attributes dicts.

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

Quentame committed:
Add icon to Plex sensor (#30172)

bachya reviewed (approved):
I would make the breaking change description more descriptive – maybe something like:

>Users will need to update any automations that utilize fan speed values – specifically:
>
>* `Quiet -> Silent`
>* `Balanced -> Standard`
>* `Turbo -> Medium`
>* `Max -> Turbo`

Can merge with this alteration and when CI passes.

springstan committed:
Add cast to state of Dyson Air Quality Sensor (#30100)

Quentame committed:
Add icons to Freebox sensors (#30132)

tiagofreire-pt commented:
> 
> 
> I would make the breaking change description more descriptive – maybe something like:
> 
> > Users will need to update any automations that utilize fan speed values – specifically:
> > 
> > * `Quiet -> Silent`
> > * `Balanced -> Standard`
> > * `Turbo -> Medium`
> > * `Max -> Turbo`
> 
> Can merge with this alteration and when CI passes.

I agree, sorry. 

This could be integrated in the Wiki, as this may be a massive breaking change for automations.

Latter on I'll add this.

Thanks.

tiagofreire-pt commented:
Docs with PR https://github.com/home-assistant/home-assistant.io/pull/11523.

gtdiehl committed:
Upgrade envoy_reader to 0.11.0 (#30179)

P-Verbrugge committed:
Updated formatting of total_blocks value (#30170)

The number of total blocks is always a round number. There can't be .1 or .11 blocks for example. The output is now always formatted with two decimals that are always 00.

heckler committed:
New date_time_utc display option added to the time_date sensor platform (#30158)

alandtse committed:
Enable config flow for Tesla (#28744)

* build: bump teslajsonpy to 0.2.0

* Remove tests

* feat: add config flow

* feat: add async

* perf: convert unnecessary async calls to sync

* feat: add charger voltage and current sensor

* feat: add options flow

* build: bump teslajsonpy to 0.2.0

* Remove icon property

* Revert climate mode change

* Remove charger sensor

* Simplify async_setup_platform

* Update homeassistant/components/tesla/sensor.py

Co-Authored-By: Paulus Schoutsen <paulus@home-assistant.io>

* Update homeassistant/components/tesla/binary_sensor.py

Co-Authored-By: Paulus Schoutsen <paulus@home-assistant.io>

* Address requested changes

* Fix pylint error

* Address requested changes

* Update codeowners

* Fix pylint error

* Address requested changes

* Address requested change

* Remove unnecessary check for existing config entry

* Load scan_interval in async_setup_entry

* Include coverage of config_flow

* Add tests for full coverage

* Address requested test changes

* Remove unnecessary init lines

* Remove unnecessary init

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

springstan commented:
@tiagofreire-pt could you please update the tests accordingly to this change? Thanks :)

dmulcahey committed:
Protect against bad data stored in ZHA (#30183)

homeassistant committed:
[ci skip] Translation update

tiagofreire-pt commented:
> 
> 
> @tiagofreire-pt could you please update the tests accordingly to this change? Thanks :)

Done. But I can't see why those 2 are failing constantly.

springstan commented:
@tiagofreire-pt you will need to push those changes as well so that the tests are succeeding again :)

bachya commented:
@tiagofreire-pt You may already know this, but you've brought in several unintended commits that relate to other files; you'll need to rebase and remove those commits.

frenck commented:
@tiagofreire-pt Small and friendly ping, could you rebase this PR and clean it up?

tiagofreire-pt committed:
Merge branch 'dev' into patch-1

tiagofreire-pt commented:
Sorry, but I couldn't solve some weird errors on rebasing my fork. Patched in another branch and submitted the new PR #30614.
